staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -778,8 +778,9 @@ const (
 	// This label represents supported cpu features on the node
 	CPUFeatureLabel = "cpu-feature.node.kubevirt.io/"
 	// This label represents supported cpu models on the node
-	CPUModelLabel = "cpu-model.node.kubevirt.io/"
-	CPUTimerLabel = "cpu-timer.node.kubevirt.io/"
+	CPUModelLabel                    = "cpu-model.node.kubevirt.io/"
+	AcceptedCPUModelInMigrationLabel = "cpu-model-migration.node.kubevirt.io/"
```

Comment:
Hey,
i don't think that `SupportedHostModelCPULabel` reflect the Purpose of this variable because the `host-model` cpuModel might not be supported and yet a we can have `AcceptedCPUModelInMigrationLabel` for the host-model in one of the nodes.
This label exist in order to chose the right node only when migrating.

Reply:
`AcceptedCPUModelInMigrationLabel` sounds like it applies for all migrations, but IIUC it applies only to host-models, therefore I think it's misleading. It somehow needs to reflect the fact that a certain CPU model is supported for host-model VMs IMHO

Reply:
i modified the name.
Thanks!